src/main/kotlin/uk/gov/justice/digital/hmpps/locationsinsideprison/resource/OperationalCapacityResource.kt

```diff
@@ -0,0 +1,108 @@
+ * @author marcus.aleman
+ *
+ *  Methods : GET & POST
+ * {
+ *   "signedOperationCapacity": 342,
+ *   "approvedBy": "MALEMAN"
+ * }
+ *
+ */
+@RestController
+@Validated
+@RequestMapping("/signed-op-cap", produces = [MediaType.APPLICATION_JSON_VALUE])
+@Tag(
+  name = "Operational Capacity",
+  description = "Returns signed operational capacity data per prison.",
+)
+class OperationalCapacityResource() : EventBaseResource() {
+
+  @GetMapping("/{prisonId}")
+  @PreAuthorize("hasRole('ROLE_VIEW_LOCATIONS')") // todo Need create a New Role?
+  @ResponseStatus(HttpStatus.OK)
+  @Operation(
+    summary = "Get Operation Capacity",
+    description = "Requires role VIEW_LOCATIONS",
+    responses = [
+      ApiResponse(
+        responseCode = "200",
+        description = "Returns Operation Capacity data",
+      ),
+      ApiResponse(
+        responseCode = "401",
+        description = "Unauthorized to access this endpoint",
+        content = [Content(mediaType = "application/json", schema = Schema(implementation = ErrorResponse::class))],
+      ),
+      ApiResponse(
+        responseCode = "403",
+        description = "Missing required role. Requires the ROLE_VIEW_LOCATIONS role",
+        content = [Content(mediaType = "application/json", schema = Schema(implementation = ErrorResponse::class))],
+      ),
+      ApiResponse(
+        responseCode = "404",
+        description = "PrisonID not found",
+        content = [Content(mediaType = "application/json", schema = Schema(implementation = ErrorResponse::class))],
+      ),
+    ],
+  )
+  fun getOperationalCapacity(
+    @Schema(description = "Prison Id", example = "MDI", required = true, minLength = 3, maxLength = 5, pattern = "^[A-Z]{2}I|ZZGHI$")
+    @PathVariable
+    prisonId: String,
+  ): OperationalCapacityDto? = OperationalCapacityDto(prisonId = "MDI", approvedBy = "MALEMAN", signedOperationCapacity = 342, dateTime = LocalDateTime.now())
+
+  @PostMapping("/{prisonId}")
+  @PreAuthorize("hasRole('ROLE_MAINTAIN_LOCATIONS') and hasAuthority('SCOPE_write')")
+  @ResponseStatus(HttpStatus.CREATED)
+  @Operation(
+    summary = "Post Operation Capacity",
+    description = "Requires role ROLE_MAINTAIN_LOCATIONS",
+    responses = [
+      ApiResponse(
+        responseCode = "201",
+        description = "Returns created Operation Capacity",
+      ),
+      ApiResponse(
+        responseCode = "401",
+        description = "Unauthorized to access this endpoint",
+        content = [Content(mediaType = "application/json", schema = Schema(implementation = ErrorResponse::class))],
+      ),
+      ApiResponse(
+        responseCode = "403",
+        description = "Missing required role. Requires the ROLE_MAINTAIN_LOCATIONS role",
+        content = [Content(mediaType = "application/json", schema = Schema(implementation = ErrorResponse::class))],
+      ),
+      ApiResponse(
+        responseCode = "404",
+        description = "PrisonID not found",
```

Comment:
Don't think this will happen as your code creates a new records if one isn't found.